com.github.gyunaev.spivak.json

```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "com.github.gyunaev.spivak",
+    "command": "spivak",
+    "rename-desktop-file": "spivak.desktop",
+    "rename-icon": "spivak",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "finish-args":[
+	"--socket=x11",
+	"--socket=pulseaudio",
+	"--share=network",
+	"--device=dri"
+    ],
+    "cleanup": [
+	"/include",
+	"/lib/pkgconfig",
+	"/lib64/pkgconfig",
+	"/share/doc",
+	"/share/man",
+	"*.a",
+	"*.la"
+    ],
+    "modules":[
+	{
```

Comment:
There's a slightly different approach to cld2 over in poedit. Not a requirement but an interesting comparison: https://github.com/flathub/net.poedit.Poedit/blob/master/net.poedit.Poedit.json#L113